packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience that can be used to authenticate users on any website. How you customize the widget depends on whether Okta is hosting the widget or you are hosting the Widget.
+
+* Self-hosted Sign-in Widget: After you have [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#getting-started) into your project and configured the authentication scenarios that you want to support, you then want to customize. You can apply customizations to match your branding using CSS and JavaScript. See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget).
+
+* Okta-hosted Sign-In Widget: Okta hosts the sign-in page for your applications that appears when your applications redirect to Okta to sign users in. You can customize the Okta-hosted sign-in page using easy controls or an embedded HTML editor that are provided on the **Custom Signin** tab of the **Customization** page in the Developer Console. See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget).
```

Comment:
```suggestion
* Okta-hosted Sign-In Widget: Okta hosts the sign-in page for your applications that appears when your applications redirect to Okta to sign users in. You can customize the Okta-hosted sign-in page using easy controls or an embedded HTML editor that is provided on the **Custom Sign In** tab of **Customization** > **Signin Page** in the Developer Console. See [Styling the Okta-hosted Sign-in Widget](#styling-the-Okta-hosted-sign-in-widget).
```

Changed to match text in developer console. It's a shame that Customization > Signin Page is not Customization > Sign In Page.